x/tss/keeper/keeper.go

```diff
@@ -225,6 +225,56 @@ func (k Keeper) GetMaxSimultaneousSignShares(ctx sdk.Context) int64 {
 	return shares
 }
 
+// returns the signed blocks window to be considered when calculating the missed blocks percentage
+func (k Keeper) getSignedBlocksWindow(ctx sdk.Context) int64 {
+	var window int64
+	k.params.Get(ctx, types.KeySignedBlocksWindow, &window)
+
+	return window
+}
+
+// HasMissedTooManyBlocks returns true if the given validator address missed too many blocks within
+// the block window specifiec by this module
+func (k Keeper) HasMissedTooManyBlocks(ctx sdk.Context, address sdk.ConsAddress) bool {
+	missedBlocks, ok := k.getMissedBlocksPercent(ctx, address)
+	if !ok {
+		return false
+	}
+	maxMissedPerWindow := k.GetMaxMissedBlocksPerWindow(ctx)
+
+	return missedBlocks.GTE(maxMissedPerWindow)
```

Comment:
I continued using`>=` because it was what we used before and I didn't want to change too much the behavior, but I also agree that we should probably use `>` instead. @sammy1991106 @haiyizxx @cgorenflo thoughts?

Reply:
If we would like to change it, now's probably a good opportunity.

Reply:
done